src/main/java/edu/oregonstate/cs361/battleship/Easy.java

```diff
@@ -4,4 +4,27 @@
  * Created by Keana on 3/15/2017.
  */
 public class Easy extends Mode {
+
+    public Easy(){
+       //nothing to construct
+    }
+    /*
+        Easy mode hard codes the placement of the computer's ships.
+     */
+    public void placeShips(Military computer_aircraftCarrier,  Military computer_battleship,  Military computer_submarine, Civilian computer_Dinghy,  Civilian computer_Clipper){
+

```

Comment:
This code looks good. It should hard code the locations of the ships appropriately.